### PR TITLE
W-10984998 duplicate class attributes

### PR DESCRIPTION
**Pull request by sanderson-sfdc**

### CODE REVIEW

**Comment by sanderson-sfdc** [comment]
Dan found this; see https://github.com/mulesoft/docs-site-ui/pull/125 but master had diverged too much from his fork to merge it.

**Review by cheungaryk — APPROVED** [review]

File: src/partials/connectors/connectors-table.hbs [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <button class="button js-connector-menu-trigger">
       <svg class="svg" width="16" height="16"><use xlink:href="#icon-kabob"/></svg>
     </button>
-    <div class="js-connector-menu" class="hidden">
+    <div class="js-connector-menu hidden">
```
Comment (mojavelinux) [inline src/partials/connectors/connectors-table.hbs:22]: This is missing a closing double quote.

**Comment by mojavelinux** [comment]
> Dan found this; see https://github.com/mulesoft/docs-site-ui/pull/125 but master had diverged too much from his fork to merge it.

Please don't hesitate to ask me to rebase an MR. I will always do it as soon as I can after I see the request, which is usually immediately.

**Comment by sanderson-sfdc** [comment]
Totally up to you @mojavelinux; I thought I was trying to save you some effort.  Do you want to rebase it or shall we continue with this one?

File: src/partials/connectors/connectors-table.hbs (hunk 1 above)
Comment (sanderson-sfdc) [reply src/partials/connectors/connectors-table.hbs:22]: It is?  class = open-quote text closed-quote

**Comment by mojavelinux** [comment]
You can proceed with this one (once the missing double quote is added). I just wanted to let you know what I'm willing to do. Rebasing an MR is very easy for me, so you aren't adding extra work by requesting it.

File: src/partials/connectors/connectors-table.hbs (hunk 1 above)
Comment (mojavelinux) [reply src/partials/connectors/connectors-table.hbs:22]: Sorry about that. I misread it. I though you were adding the hidden attribute, but you are adding the hidden class. Both ways work, but this is fine as is.

**Comment by mojavelinux** [comment]
Yep, you are right. All good!

**Pull request merged**